lib/configure-map.js

```diff
@@ -135,10 +135,24 @@ module.exports = function (original, length, options) {
 			return conf.delete(arg);
 		};
 	}
+	extGet = function () {
+		var id, args = arguments;
+		if (resolve) args = resolve(args);
+		id = get(args);
+		return cache[id];
+	};
+	extHas = function () {
+		var id, args = arguments;
+		if (resolve) args = resolve(args);
+		id = get(args);
+		return conf.has(id);
+	};
 	defineProperties(memoized, {
 		__memoized__: d(true),
 		delete: d(extDel),
-		clear: d(conf.clear)
+		clear: d(conf.clear),
+		get: d(extGet),
```

Comment:
Yes, in v1, with `get` and `has`, you will always get resolved value, so final value in case of async and promise modes.
I also wrote an updated API proposal already -> https://github.com/medikoo/memoizee/issues/73 (although it doesn't explain that detail)